jolly-jellyfish/src/page_maker/templates/page_maker/base.html

```diff
@@ -0,0 +1,41 @@
+{% load static %}
+<!doctype html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+
+    {% if webpage %}
+      <title>{{ webpage.user_title }}</title>
+    {% else %}
+      <title>Early Internet Page Maker</title>
+    {% endif %}
```

Comment:
Pretty sure we aren't indenting inside if blocks...

Reply:
Maybe we should though